### ISSUE TITLE
Handlebars 2.0 support for argument passing into helpers and partial blocks

### ISSUE THREAD

**Issue by ericlowry** [fxbois/web-mode#384]
Handlebars now supports passing partial block arguments, which confuse web-mode's syntax highlighting.

Example 1:  {{ > partialBlock v1="String Constant" v2=varName v3=varName.attrName v4=var1num }}

Example 2:  {{#blockHelper v1="String Constant" v2=varName v3=varName.attrName v4=var1num }}

v1 and v2 work fine.  Numeric constants work fine as well, like v1=100.

v3's highlighting is confusing, I think both varName and attrName should be the same color.

v4 is broken.

To recap; the highlighting issues occur when the identifier contains a number like var1num or when an object includes an attribute reference like varName.attr.
